extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1285,12 +1286,22 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
         'date=${timestamp}&' +
         'ampid=${clientId(_parsely_visitor)}',
       'pageview': '${basePrefix}&action=pageview',
+      'heartbeat': '${basePrefix}&action=heartbeat' +
+      '&tt=${totalEngagedTime}&inc=${incrementalEngagedTime}',
     },
     'triggers': {
       'defaultPageview': {
         'on': 'visible',
         'request': 'pageview',
       },
+      'heartbeat': {
+        'on': 'timer',
+        'timerSpec': {
+          'interval': 5,
+          'name': 'parsely-js',
```

Comment:
I believe you want
```
'vars': {
...
'name': 'parsely-js,
}
```
Then 
```
'heartbeat': '${basePrefix}&action=heartbeat&tt=${totalEngagedTime}&inc=${incrementalEngagedTime(${name}}',
```

Or save the `name` var by using `inc=${incrementalEngagedTime(parsely-js)}`